trusted_shuffler/trusted_shuffler/src/tests.rs

```diff
@@ -52,30 +61,30 @@ fn generate_response(request: &TrustedShufflerRequest) -> TrustedShufflerRespons
 
 // Generates request which will be dropped by the test backend, because we indicate it has to be
 // dropped in the request itself.
-fn generate_dropped_request() -> TrustedShufflerRequest {
+fn generate_dropped_request() -> EncryptedRequest {
```

Comment:
Why do we need requests that'll be dropped by the backend? Is it for testing?
I think testing code shouldn't be a part of the trusted part of the trusted shuffler

Reply:
It is for testing and pre-existing. I'd leave it for this PR.